romana-install/config.devstack.yml

```diff
@@ -17,3 +17,7 @@
 - hosts: devstack_computes
   roles:
     - stack/devstack/install-compute
+
+- hosts: devstack_controller
+  roles:
+    - stack/devstack/postinstall
```

Comment:
It's things that need to be done once to prepare the system (creating network, adding keypair etc). It doesn't need doing on compute nodes. Any node would do, but controller seemed more sensible.